Entity/Buff.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SharpGame.Entity
+{
+    public abstract class Buff
+    {
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public abstract void Add(Entity Target);
+        public abstract void Remove(Entity Target);
```

Comment:
Make 'Target' lowercase in the parameter list for both Add and Remove. Also, add a blank line between methods and properties in classes.